call/contract.go

```diff
@@ -27,12 +28,14 @@ type Method struct {
 type ContractBuilder interface {
 	WithClient(ethClient *ethclient.Client) ContractBuilder
 	AtAddress(contractAddress string) ContractBuilder
-	AddMethod(signature string) ContractBuilder
+	AddMethod(signature string) *Contract
 	Abi() abi.ABI
-	Build() *contract
+	Build() *Contract
+	DefaultConfig(chain Chain) *Contract
+	CustomConfig(config ChainConfig) *Contract
```

Comment:
I think that is a convenient idea. However, It will be easier when we can simplify this couple of the functions. So how do you think about the mechanism below:
- We always have Ethereum as the default chain when we call NewContractBuilder()
- Add only `CustomConfig(config ChainConfig) *Contract` as an interface to support change config to other chains 

---

It is not much important but I like `WithChainConfig(config ChainConfig)` than `CustomConfig(config ChainConfig) *Contract` 😆

Reply:
I like the idea of set Ethereum as the default chain when we call NewContractBuilder().
I want a function that sets a default configuration for each of the supported chains. Do you have suggestions for this with only one `CustomConfig` function? @himitsuko

Reply:
It thinking about somethings like `NewContractBuilder().Ethereum()` for easier initialize.
However it sounds bad idea 😄 

---

Actually I only want the default is the default. And I don't like to config any more when we use the default.
Otherwise, when we need to config something to override the default `With[Thing]Config()` is simple to understand. I think so.

And only one `CustomConfig` function is better for the future maintainance when we have only one interface on every like of code.

How do you think when we mixing the implemented of this two comments?
- https://github.com/depocket/multicall-go/pull/2#discussion_r896379243
- https://github.com/depocket/multicall-go/pull/2#discussion_r896381990

Default, we initial any default value in NewContractBuilder() like default ethereum chain, default contract address,..etc
And the next, we have `WithChainConfig(config ChainConfig)` or `CustomChainConfig(config ChainConfig)` to override the default chain.
Then we pass the `Chains.Ethereum` or `Chains['ethereum']` as the param's value to easier to get the config of the supported chains.

Reply:
>I want a function that sets a default configuration for each of the supported chains

I think we call `Chains.Ethereum` or `Chains['ethereum']` is make sense for the default for each of the supported chains

Reply:
Good idea. I will update it like this.